### PR TITLE
Add ConversationResourceResponce for create_conversation api

### PR DESCRIPTION
**Pull request by Dashenok**
Create conversation API https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#create-conversation

ConversationResourceResponce object https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#conversationresourceresponse-object

### CODE REVIEW

File: lib/models/conversation_resource_response.ex [hunk 1]
```diff
@@ -0,0 +1,34 @@
+defmodule ExMicrosoftBot.Models.ConversationResourceResponse do
+  @moduledoc """
+  Microsoft bot conversation API resource response
+  """
+
+  @derive [Poison.Encoder]
+  defstruct [:id, :activityId]
```
Comment (yondermon) [inline lib/models/conversation_resource_response.ex]:
```suggestion
  defstruct [:id, :activityId, :serviceUrl]
```

File: lib/models/conversation_resource_response.ex [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  @type t :: %ExMicrosoftBot.Models.ResourceResponse{
```
Comment (yondermon) [inline lib/models/conversation_resource_response.ex]:
```suggestion
  @type t :: %ExMicrosoftBot.Models.ConversationResourceResponse{
```

File: lib/models/conversation_resource_response.ex [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    id: String.t,
+    activityId: String.t
```
Comment (yondermon) [inline lib/models/conversation_resource_response.ex]:
```suggestion
    activityId: String.t,
    serviceUrl: String.t
```

File: lib/models/conversation_resource_response.ex [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  @doc """
+  Decode a map into `ExMicrosoftBot.Models.ResourceResponse`
+  """
+  @spec parse(map) :: {:ok, ExMicrosoftBot.Models.ResourceResponse.t}
```
Comment (yondermon) [inline lib/models/conversation_resource_response.ex]:
```suggestion
  @spec parse(map) :: {:ok, ExMicrosoftBot.Models.ConversationResourceResponse.t}
```

File: lib/models/conversation_resource_response.ex [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  def parse(param) when is_map(param) do
+    {:ok, Poison.Decode.decode(param, as: decoding_map())}
+  end
+
+  @doc """
+  Decode a string into `ExMicrosoftBot.Models.ResourceResponse`
+  """
+  @spec parse(String.t) :: ExMicrosoftBot.Models.ResourceResponse.t
```
Comment (yondermon) [inline lib/models/conversation_resource_response.ex]:
```suggestion
  @spec parse(String.t) :: ExMicrosoftBot.Models.ConversationResourceResponse.t
```

File: lib/models/conversation_resource_response.ex [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  @doc """
+  Decode a map into `ExMicrosoftBot.Models.ResourceResponse`
+  """
+  @spec parse(map) :: {:ok, ExMicrosoftBot.Models.ResourceResponse.t}
+  def parse(param) when is_map(param) do
+    {:ok, Poison.Decode.decode(param, as: decoding_map())}
+  end
+
+  @doc """
+  Decode a string into `ExMicrosoftBot.Models.ResourceResponse`
```
Comment (yondermon) [inline lib/models/conversation_resource_response.ex]:
```suggestion
  Decode a string into `ExMicrosoftBot.Models.ConversationResourceResponse`
```

File: lib/models/conversation_resource_response.ex [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  @derive [Poison.Encoder]
+  defstruct [:id, :activityId]
+
+  @type t :: %ExMicrosoftBot.Models.ResourceResponse{
+    id: String.t,
+    activityId: String.t
+  }
+
+  @doc """
+  Decode a map into `ExMicrosoftBot.Models.ResourceResponse`
```
Comment (yondermon) [inline lib/models/conversation_resource_response.ex]:
```suggestion
  Decode a map into `ExMicrosoftBot.Models.ConversationResourceResponse`
```

File: lib/models/conversation_resource_response.ex [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  Decode a string into `ExMicrosoftBot.Models.ResourceResponse`
+  """
+  @spec parse(String.t) :: ExMicrosoftBot.Models.ResourceResponse.t
+  def parse(param) when is_binary(param) do
+    Poison.decode!(param, as: decoding_map())
+  end
+
+  @doc false
+  def decoding_map() do
+    %ExMicrosoftBot.Models.ResourceResponse{}
```
Comment (yondermon) [inline lib/models/conversation_resource_response.ex]:
```suggestion
    %ExMicrosoftBot.Models.ConversationResourceResponse{}
```

**Review by yondermon — CHANGES_REQUESTED** [review]

File: lib/models/conversation_resource_response.ex (hunk 1 above)
Comment (Dashenok) [reply lib/models/conversation_resource_response.ex]: What if this field is absent in response? Will it cause exceptions?

Comment (yondermon) [reply lib/models/conversation_resource_response.ex]: No, it should be nil.

**Review by yondermon — APPROVED** [review]

**Pull request merged**